source/geod24/concurrency.d

```diff
@@ -1103,13 +1103,11 @@ protected:
         override void notify() nothrow
         {
             notified = true;
-            FiberScheduler.yield();
         }
 
         override void notifyAll() nothrow
         {
             notified = true;
-            FiberScheduler.yield();
```

Comment:
That completely changes the meaning of this function

Reply:
I'll change to the original code.